src/components/PreferencesModal/KeymapTab.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useCallback, useMemo } from 'react'
+import { Section, SectionHeader } from './styled'
+import { usePreferences } from '../../lib/preferences'
+import { KeymapItem, KeymapItemEditableProps } from '../../lib/keymap'
+import { useTranslation } from 'react-i18next'
+import KeymapItemSection from '../atoms/KeymapItemSection'
+import styled from '../../lib/styled/styled'
+
+const KeymapTab = () => {
+  const {
+    preferences,
+    updateKeymap,
+    removeKeymap,
+    resetKeymap,
+  } = usePreferences()
+  const { t } = useTranslation()
+
+  const keymap = useMemo(() => {
+    const keymap = preferences['general.keymap']
+    if (keymap != null) {
+      return [...keymap.entries()]
+    } else {
+      return []
+    }
+  }, [preferences])
+
+  const handleAssignNewKeymap = useCallback(
+    (
+      key: string,
+      firstShortcut: KeymapItemEditableProps,
+      secondShortcut?: KeymapItemEditableProps
+    ) => {
+      return updateKeymap(key, firstShortcut, secondShortcut)
+    },
+    [updateKeymap]
+  )
+
+  const handleRemoveKeymap = useCallback(
+    (key: string) => {
+      removeKeymap(key)
+    },
+    [removeKeymap]
+  )
+
+  const handleResetKeymap = useCallback(() => {
+    resetKeymap()
```

Comment:
Updates correctly, but below components rerenders old keymap instead of new.. maybe memo is not working correctly? Not sure - needs to be fixed that on reset UI is updated and correct shortcuts are shown. Instead, it currently updates accelerators in menus but not keymap tab shortcuts strings...

Reply:
`KeymapItemSection` had an incorrect key, so the component did not update correctly! Fixed in the latest commit.